metadata/src/main/java/org/apache/kafka/controller/QuorumControllerMetrics.java

```diff
@@ -30,14 +30,24 @@
         "kafka.controller", "ControllerEventManager", "EventQueueTimeMs", null);
     private final static MetricName EVENT_QUEUE_PROCESSING_TIME_MS = new MetricName(
         "kafka.controller", "ControllerEventManager", "EventQueueProcessingTimeMs", null);
+    private final static MetricName GLOBAL_TOPIC_COUNT = new MetricName(
+        "kafka.controller", "ReplicationControlManager", "GlobalTopicCount", null);
+    private final static MetricName GLOBAL_PARTITION_COUNT = new MetricName(
+        "kafka.controller", "ReplicationControlManager", "GlobalPartitionCount", null);
 
     private volatile boolean active;
+    private volatile int topics;
+    private volatile int partitions;
```

Comment:
I think it would be preferrable to call this "globalPartitionCount" and call the gauge "globalPartitionCountGauge" or something similar.